katz/src/main/kotlin/katz/data/Either.kt

```diff
@@ -61,8 +61,8 @@ sealed class Either<out A, out B> : EitherKind<A, B> {
      *
      * Example:
      * ```
-     * Left("left").swap()   // Result: Right("left")
-     * Right("right").swap() // Result: Left("right")
+     * Left("left").reverse()   // Result: Right("left")
+     * Right("right").reverse() // Result: Left("right")
      * ```
      */
     fun swap(): Either<B, A> =
```

Comment:
The name of the method is still swap. The documentation should be reverted to say swap too. Let's keep revert only for tuples!